tmt-app/src/Components/CustomCard.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Container,
+} from "@mui/material";
+import { ReactComponent as EmotionalIntelligence } from "../Images/icons/emotional_intelligence.svg";
+import { Link } from "react-router-dom";
+
+function CustomCard({ dummyData }) {
+  const career = "education";
+  return (
+    <Link
+      rel="noopener noreferrer"
+      to={`/career/${career}`}
```

Comment:
Did this get tested yet?

Reply:
yes, I was able to get the info I wanted after

Reply:
That is expected because I was passing a constant variable for testing purposes, and I was planning to update it once the data file is complete. I will make changes to retrieve from the mock data for now.